onepassword/resource_onepassword_item.go

```diff
@@ -9,16 +9,60 @@ import (
 	"github.com/1Password/connect-sdk-go/onepassword"
 	"github.com/hashicorp/go-uuid"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 )
 
+const (
+	itemUUIDDescription  = "The UUID of the item. Item identifiers are unique within a specific Vault."
```

Comment:
I'd use the capital Vault when specifically talking about a Vault object, not vaults at large.

```suggestion
	itemUUIDDescription  = "The UUID of the item. Item identifiers are unique within a specific vault."
```